build/secrets/.secrets.baseline

```diff
@@ -1711,7 +1711,7 @@
       {
         "type": "Base64 High Entropy String",
         "filename": "src/vs/workbench/contrib/webview/browser/pre/index.html",
-        "hashed_secret": "8f848d45aabb6a8b8da85a44943af7f622b9ade3",
+        "hashed_secret": "8e7109befbc4b291b584a623b3f203fb3a7c7fea",
```

Comment:
missed updating the secrets baseline for this entry in https://github.com/posit-dev/positron/pull/8770